packages/@ourworldindata/types/src/dbTypes/Charts.ts

```diff
@@ -1,41 +1,13 @@
-import { JsonString } from "../domainTypes/Various.js"
-import { GrapherInterface } from "../grapherTypes/GrapherTypes.js"
-
 export const ChartsTableName = "charts"
 export interface DbInsertChart {
-    config: JsonString
+    configId: string
     createdAt?: Date
```

Comment:
Binary fields become node Buffer values
```suggestion
    configId: Buffer
```